drivers/bridge/labels.go

```diff
@@ -0,0 +1,27 @@
+package bridge
+
+const (
+	// BridgeName label for bridge driver
+	BridgeName = "io.docker.network.bridge.name"
+
+	// EnableIPTables label for bridge driver
+	EnableIPTables = "io.docker.network.bridge.enable_iptables"
+
+	// EnableIPMasquerade label for bridge driver
+	EnableIPMasquerade = "io.docker.network.bridge.enableipmasquerade"
```

Comment:
This looks inconsistent with the other "enable" labels, should be `enable_ipmasquerade` (or even `enable_ip_masquerade`?).

Given that a "reverse domain" notation is used as convention, I think dashes (`'-'`) in stead of underscores may be preferred as separator between words